Error when running npm run watch

I'm on Windows. Installed jupyter with anaconda. And installed jupyterlab using source from git.
I'm trying to modify some typescript and less code and checking whether my changes are reflected in jupyterlab.

I get the following error when running `npm run watch`:

```
...\jupyterlab>npm run watch

> jupyterlab@0.2.0 watch ...\jupyterlab
> watch 'npm run build:all' src --wait 10

> Watching run
> Watching build:all'
> Watching src
...\jupyterlab\node_modules\watch\main.js:73
    if (err) throw err;
             ^

Error: ENOENT: no such file or directory, stat '...\jupyterlab\run'
    at Error (native)

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "...\\nvm\\v6.4.0\\node.exe" "...\\nvm\\v6.4.0\\node_modules\\npm\\bin\\npm-cli.js" "run" "wat
ch"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! jupyterlab@0.2.0 watch: `watch 'npm run build:all' src --wait 10`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterlab@0.2.0 watch script 'watch 'npm run build:all'
src --wait 10'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyterlab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     watch 'npm run build:all' src --wait 10
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyterlab
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyterlab
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     ...\jupyterlab\npm-debug.log
```

When  run npm run build, it works though (although the changes aren't reflected on the jupyterlab page):
```
...\jupyterlab>npm run build

> jupyterlab@0.2.0 build ...\jupyterlab
> npm run build:css && npm run build:src


> jupyterlab@0.2.0 build:css ...\jupyterlab
> concurrently "npm run build:css:base" "npm run build:css:theme"

[0]
[0] > jupyterlab@0.2.0 build:css:base ...\jupyte
rlab
[0] > lessc --include-path=node_modules src/basestyle/index.less lib/basestyle/i
ndex.css
[0]
[1]
[1] > jupyterlab@0.2.0 build:css:theme ...\jupyt
erlab
[1] > lessc  --include-path=node_modules src/default-theme/index.less lib/defaul
t-theme/index.css
[1]
[0] npm run build:css:base exited with code 0
[1] npm run build:css:theme exited with code 0

> jupyterlab@0.2.0 build:src ...\jupyterlab
> tsc --project src && node scripts/copyfiles.js
```

--- a/examples/console/package.json
+++ b/examples/console/package.json
@@ -6,7 +6,7 @@
     "clean": "rimraf build && rimraf node_modules",
     "postinstall": "node ../../scripts/dedupe.js",
     "update": "rimraf node_modules/jupyterlab && npm install",
-    "watch": "watch 'npm run update && npm run build' ../../src src --wait 10"
+    "watch": "watch \"npm run update && npm run build\" ../../src src --wait 10"
   },
   "dependencies": {
     "jupyter-js-services": "^0.18.0",

--- a/examples/filebrowser/package.json
+++ b/examples/filebrowser/package.json
@@ -6,7 +6,7 @@
     "clean": "rimraf build && rimraf node_modules",
     "postinstall": "node ../../scripts/dedupe.js",
     "update": "rimraf node_modules/jupyterlab && npm install",
-    "watch": "watch 'npm run update && npm run build' ../../src src --wait 10"
+    "watch": "watch \"npm run update && npm run build\" ../../src src --wait 10"
   },
   "dependencies": {
     "jupyter-js-services": "^0.18.0",

--- a/examples/lab/package.json
+++ b/examples/lab/package.json
@@ -6,7 +6,7 @@
     "clean": "rimraf build && rimraf node_modules",
     "postinstall": "node ../../scripts/dedupe.js",
     "update": "rimraf node_modules/jupyterlab && npm install",
-    "watch": "watch 'npm run update && npm run build' ../../src --wait 10"
+    "watch": "watch \"npm run update && npm run build\" ../../src --wait 10"
   },
   "dependencies": {
     "es6-promise": "^3.1.2",

--- a/examples/notebook/package.json
+++ b/examples/notebook/package.json
@@ -6,7 +6,7 @@
     "clean": "rimraf build && rimraf node_modules",
     "postinstall": "node ../../scripts/dedupe.js",
     "update": "rimraf node_modules/jupyterlab && npm install",
-    "watch": "watch 'npm run update && npm run build' ../../src src --wait 10"
+    "watch": "watch \"npm run update && npm run build\" ../../src src --wait 10"
   },
   "dependencies": {
     "jupyter-js-services": "^0.18.0",

--- a/examples/terminal/package.json
+++ b/examples/terminal/package.json
@@ -6,7 +6,7 @@
     "clean": "rimraf build && rimraf node_modules",
     "postinstall": "node ../../scripts/dedupe.js",
     "update": "rimraf node_modules/jupyterlab && npm install",
-    "watch": "watch 'npm run update && npm run build' ../../src src --wait 10"
+    "watch": "watch \"npm run update && npm run build\" ../../src src --wait 10"
   },
   "dependencies": {
     "jupyter-js-services": "^0.18.0",

--- a/package.json
+++ b/package.json
@@ -71,9 +71,9 @@
     "test:debug": "npm run build:test && karma start --browsers=Chrome --singleRun=false --debug=true test/karma.conf.js",
     "test:firefox": "npm run build:test && karma start --browsers=Firefox test/karma.conf.js",
     "test:ie": "npm run build:test && karma start --browsers=IE test/karma.conf.js",
-    "watch": "watch 'npm run build:all' src --wait 10",
-    "watch:src": "watch 'npm run build' src --wait 10",
-    "watch:test": "watch 'npm run build && npm test' src test/src --wait 10"
+    "watch": "watch \"npm run build:all\" src --wait 10",
+    "watch:src": "watch \"npm run build\" src --wait 10",
+    "watch:test": "watch \"npm run build && npm test\" src test/src --wait 10"
   },
   "repository": {
     "type": "git",

